packages/headless/src/modules/brands/validators/brand.create.validator.ts

```diff
@@ -0,0 +1,21 @@
+import * as Joi from 'joi';
+
+import { Brand } from './../../../entity/brand';
+
+export const BrandCreateSchema = Joi.object<Brand>({
+    id: Joi.string(),
+    info:{
```

Comment:
when object() used then it assumes the schema type to be any, but as we want it to be of type Brand, so thats why it can and also cannot be used if we are open to open inputs